### DISCUSSION TITLE
Server connection error

### DISCUSSION THREAD

**Discussion by moviemanrico** [ConsenSys-archive/truffle#6221] [General Q&A]
When I try to connect, I m getting this message:

' ' '
error sending request for url (http://127.0.0.1:7545/): error trying to connect: tcp connect error: Connection refused (os error 111)
' ' '

**Comment by MadelineMurray** [comment]
hi @moviemanrico - Truffle is being sunset. For more information and migration options, click [here](https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat?utm_source=github&utm_medium=referral&utm_campaign=2023_Sep_truffle-sunset-2023_announcement_).
